linkerd/http/retry/src/replay.rs

```diff
@@ -141,18 +141,11 @@ impl<B: Body> ReplayBody<B> {
     ///
     /// If this is true, the body is now empty, and the request should *not* be
     /// retried with this body.
-    pub fn is_capped(&self) -> bool {
+    pub fn is_capped(&self) -> Option<bool> {
```

Comment:
:point_up: this is the core of the proposed change.

if there's an outstanding clone still being polled, return `None`.